website/docs/guides/markdown-features/markdown-features-code-blocks.mdx

```diff
@@ -788,7 +796,12 @@ npm install @docusaurus/remark-plugin-npm2yarn
 ```
 ````
 
-Using the `{sync: true}` option would make all tab choices synced. Because the choice is stored under the same namespace `npm2yarn`, different `npm2yarn` plugin instances would also sync their choices.
+#### Configuration {#npm2yarn-remark-plugin-configuration}
+
+| Option | Type | Default | Description |
+| --- | --- | --- | --- |
+| `sync` | `boolean` | `false` | Whether to sync the selected converter across all code blocks. |
+| `converters` | `array` | `'yarn'`, `'pnpm'` | The list of converters to use. The order of the converters is important, as the first converter will be used as the default choice. |
```

Comment:
The package readme file needs an update too:

https://github.com/facebook/docusaurus/blob/f8edeb081643e2b5b8096f19d178e31e15e1f4b8/packages/docusaurus-remark-plugin-npm2yarn/README.md?plain=1#L61-L65

Reply:
👍 

good place to mention custom converters too